Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -128,15 +129,30 @@ namespace AZ::SceneAPI::Behaviors
         ManifestAction action,
         [[maybe_unused]] RequestingApplication requester)
     {
-        // this toggle makes constructing default mesh groups and a prefab optional
+        using namespace SceneAPI::Events;
```

Comment:
suggestion: It's generally recommended/preferred to about `using namespace` directives (due to the name lookup rule issues it can cause) and instead use a `using` declaration such as `using SceneAPI::Events;` (or whatever type(s) is/are needed (or use a namespace alias if that makes things cleaner)